src/h3lib/lib/algos.c

```diff
@@ -90,9 +91,8 @@ int H3_EXPORT(maxKringSize)(int k) {
  */
 void H3_EXPORT(kRing)(H3Index origin, int k, H3Index* out) {
     int maxIdx = H3_EXPORT(maxKringSize)(k);
-    int* distances = calloc(maxIdx, sizeof(int));
+    int distances[maxIdx];
```

Comment:
I was a little concerned this does not initialize `distances`. It might be that `distances` cannot be used before initializing. Not sure if you want to add a memset here for clarity.

Reply:
Good catch. Was almost certain I followed them all with a `memset`.

Reply:
Fixed.